assets/src/edit-story/karma/fixture/containers/canvas.js

```diff
@@ -125,4 +125,8 @@ class Frame extends Container {
   constructor(node, path) {
     super(node, path);
   }
+
+  get textContent() {
```

Comment:
Since it's the `innerHTML` let's call it that?

Reply:
But it's innerHTML of a certain child, that's "somewhere" in the DOM. The naming reflects, that is the contents of this frame *if* it's a text frame. If invoked for any other frame, it would (and should) be `null`. Is that fair?

Reply:
Ok, got it. But kind of stresses the confusion point :) What about `textContentHtml()` to split the difference?

Reply:
Fixed in 1306a7e5d.